client/requirements.txt

```diff
@@ -11,3 +11,4 @@ redis>=4.5.5
 opentelemetry-api>=1.18.0
 opentelemetry-sdk>=1.18.0
 opentelemetry-exporter-otlp-proto-grpc>=1.18.0
+s3fs>=2023.5.0
```

Comment:
I was taking a look and it seems `s3fs` doesn't use `sem-ver`. Should we pin the version? It seems each version doesn't add too much things but just thinking in loud.

Reply:
I'm fine either way (originally had it pinned but switched based on @IceKhan13 's [review](https://github.com/Qiskit-Extensions/quantum-serverless/pull/624#discussion_r1214346135))

Reply:
Oh, I didn't see it, sorry. I don't have a strong opinion so @IceKhan13 if you think it's fine let us know and we can resolve this conversation.